assets/scripts/routes/common.js

```diff
@@ -14,38 +12,42 @@ export default {
 					sections[i].className += ' focusable';
 				}
 
-				// If there is a '#' in the URL (someone linking directly to a page with an anchor), go directly to that area and focus is
-				// Thanks to WebAIM.org for this idea
-				if ( document.location.hash && document.location.hash !== '#' ) {
-					let anchorUponArrival = document.location.hash;
-					setTimeout( function () {
-						$( anchorUponArrival ).scrollTo( { duration: 1500 } );
-						$( anchorUponArrival ).trigger( 'focus' );
-					}, 100 );
+				// Scroll to anchor if there's a hash in the URL & anchor exists on the page
+				if (document.location.hash && document.location.hash !== '#') {
+					const anchorUponArrival = document.location.hash;
+					setTimeout(() => {
+						const $anchorElement = $(anchorUponArrival);
+						if ($anchorElement.length) {
+							$anchorElement.scrollTo({ duration: 1500 });
+							$anchorElement.trigger('focus');
+						} else {
+							console.warn(`Anchor element "${anchorUponArrival}" not found in the document.`);
+						}
+					}, 100);
 				}
 			} );
 			$( '.js-header-nav-toggle' ).on( 'click', event => {
 				event.preventDefault();
 				$( '.header__nav' ).toggleClass( 'header__nav--active' );
 			} );
 		} );
-		// Props to Dave Rupert: https://daverupert.com/2017/11/happier-html5-forms/
-		const inputs = document.querySelectorAll( 'input, textarea' );
 
+		// Check form field validity when focus changes
+		const inputs = document.querySelectorAll( 'input, textarea' );
 		inputs.forEach( input => {
-			input.addEventListener(
-				'invalid',
-				event => {
-					input.classList.add( 'error' );
-				},
-				false
-			);
+			input.addEventListener('invalid', () => {
+					input.classList.add('error');
+				});
```

Comment:
we have a linting issue here it seems

Reply:
I'm not seeing any locally. What's the issue? 
![Screenshot 2024-12-16 155820](https://github.com/user-attachments/assets/51a509f4-5ab4-4a46-b915-7e8901d0320c)